### DISCUSSION TITLE
When will firefly-iii:latest reflect version 5.5.3?

### DISCUSSION THREAD

**Discussion by Echelon101** [firefly-iii/firefly-iii#4590] [Firefly III] [answered]
Hello There,

due to the error in billing, I had the need to update to 5.5.3. I've deployed firefly via docker-compose and firefly-iii:latest does not contain 5.5.3. Can anyone tell me when this will happen?

Best regards 

Timo

**Comment by JC5** [comment, marked as answer]
Probably tomorrow when I run the build.

**Reply by Echelon101** [reply]
Thank you for the information. I look forward to it. Thank you for fixing the issue so fast.
